TS03-Subset-Data-and-No-Data-Values.Rmd

```diff
@@ -190,11 +187,12 @@ have NA values in our other variables.
 When we encounter NA or missing values (blank, NaN, -9999, etc.) in our data we
 need to decide at how to deal with them.  By default R treats NA values as 
 blanks, not as zeros.  This is good as a value of zero (no rain today) is not
-the same as lack of data (we didn't measure the rain today). 
+the same as lack of data (we didn't measure the rain today). You may have some 
+CSV files where a blank is meant as a 'zero'- be used to check the metadata!
```

Comment:
"be used to check the metadata"  change to "get used to..." or maybe "and important reason to check...".